tendermint/src/time.rs

```diff
@@ -118,6 +118,16 @@ impl Time {
         timestamp::to_rfc3339_nanos(self.0.assume_utc())
     }
 
+    /// Return an unix timestamp with seconds.
+    pub fn to_timestamp(&self) -> i64 {
+        self.0.assume_utc().unix_timestamp()
+    }
+
+    /// Return an unix timestamp with nanoseconds.
+    pub fn to_timestamp_nanos(&self) -> i128 {
+        self.0.assume_utc().unix_timestamp_nanos()
+    }
```

Comment:
I think these would make more sense if you kept the inner names: `unix_timestamp` and `unix_timestamp_nanos`

"Timestamp" doesn't necessarily imply seconds from the Unix epoch to me

Reply:
Okay. thanks for reply I'll update it In a few minutes.